arangod/Graph/Steps/ClusterProviderStep.h

```diff
@@ -1,3 +1,5 @@
+#pragma once
```

Comment:
OOHH!

Here is the Licencse disclaimer missing!

Reply:
Added.